### PR TITLE
fix: scroll

### PR DESCRIPTION
**Pull request by RomanistNLX**
scroll to the last child of chat when opened

### CODE REVIEW

**Comment by RomanistNLX** [comment]
@peterszerzo adjusted as discussed

File: packages/react/src/index.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export interface ChatHook {
   messagesContainerRef: Ref<HTMLDivElement>;
   waiting: boolean;
   setInputValue: (val: string) => void;
+  scrollToBottom: () => void;
```
Comment (peterszerzo) [inline packages/react/src/index.ts:19]: Perform all changes in this file in the `packages/preact/src/index.ts` file as well. There is a comment marking at what point in the file you can just do a full-on copy-paste: https://github.com/nlxai/chat-sdk/blob/master/packages/react/src/index.ts#L3

**Review by peterszerzo — CHANGES_REQUESTED** [review]

File: packages/react/src/index.ts (hunk 1 above)
Comment (RomanistNLX) [reply packages/react/src/index.ts:19]: done

**Comment by RomanistNLX** [comment]
@peterszerzo we also tested workflow, it's working :)

**Review by peterszerzo — APPROVED** [review]

**Pull request merged**